modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -55,14 +55,102 @@ definition {
 		}
 	}
 
-	tearDown {
-		var testPortalInstance = PropsUtil.get("test.portal.instance");
+	//tearDown {
+	//	var testPortalInstance = PropsUtil.get("test.portal.instance");
+	//	if ("${testPortalInstance}" == "true") {
+	//		PortalInstances.tearDownCP();
+	//}
+	//	else {
+	//		JSONGroup.deleteGroupByName(groupName = "Partner Portal");
+	//}
+	//}
+
+	@description = "This is a test for LPS-172917. Verify that If there are multiple activities, the review page will display them as dropdown items that can be expandable"
+	@priority = "5"
+	test CanExpandAndSeeMultipleActivities {
+		task ("Given I go to add a MDF Request") {
+			PRMMarketing.gotoAddNewMDF();
+		}
+
+		task ("And I fill all Goals form with my informations") {
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "Company Name 1",
+				additionalOptions = "6.x to DXP Migration/Upgrade,Migration from competitor platform",
+				campaignDescription = "Campaign Description",
+				campaignName = "Campaign Name",
+				countryName = "Global",
+				goalsOptions = "Lead generation",
+				targetMarketOptions = "Education,Energy",
+				targetRolesOptions = "Administrator");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("And I add a activity") {
+			PRMMDFRequest.newActivity(
+				activityDescription = "Activity Description",
+				activityName = "Activity Name 1",
+				budgetList = "Sponsorship Fee-2000,Room Rental-3000",
+				followUpStrategy = "test",
+				leadFollowUpList = "Other",
+				leadList = "No",
+				tactic = "Other",
+				targetOfLeads = "test",
+				typeOfActivity = "Miscellaneous Marketing");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("And I add a new activity") {
+			PRMMDFRequest.newActivity(
+				activityDescription = "Activity Description",
+				activityIndex = "1",
+				activityName = "Activity Name 2",
+				budgetList = "Sponsorship Fee-2000,Room Rental-3000",
+				followUpStrategy = "test",
+				leadFollowUpList = "Other",
+				leadList = "No",
+				tactic = "Other",
+				targetOfLeads = "test",
+				typeOfActivity = "Miscellaneous Marketing");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("When I go to Review MDF") {
+			Button.click(button = "Continue");
+		}
+
+		task ("Then Click on the first activity validate Activity name") {
+			PRMMDFRequest.click(button = "Activity Name 1");
+
+			AssertVisible(
+				key_fieldName = "Activity name",
+				key_fieldValue = "Activity Name 1",
+				locator1 = "PRMMDFRequest#SUMMARY_FIELD");
 
-		if ("${testPortalInstance}" == "true") {
-			PortalInstances.tearDownCP();
+			PRMMDFRequest.click(button = "Activity Name 1");
+
+			AssertNotVisible(
+				key_fieldName = "Activity name",
+				key_fieldValue = "Activity Name 1",
+				locator1 = "PRMMDFRequest#SUMMARY_FIELD");
```

Comment:
Hi @juliosilveira77, could you please add this assert in a new task? It can be: "Then the information about the first activity will no longer be visible"